README.md

```diff
@@ -1,32 +1,67 @@
-﻿# LoTAS Mod
-### Low Optimization Tool Assisted Speedrun Mod
+﻿# This project is on hiatus!
+We are focussing our attention to [TASmod](https://github.com/MinecraftTAS/TASmod).  
```

Comment:
"focu**s**ing" for international, also write it like this:
`We are shifting our focus to ...`